crates/rolldown/src/stages/generate_stage/compute_cross_chunk_links.rs

```diff
@@ -264,6 +263,7 @@ impl GenerateStage<'_> {
             depended_symbols.insert(self.link_output.runtime.resolve_symbol("__toCommonJS"));
             depended_symbols.insert(entry.namespace_object_ref);
           }
+          // depended_symbols.insert(self.link_output.runtime.resolve_symbol("__name"));
```

Comment:
I guess this should be done in `reference_needed_symbols`. Insert reference to `__name` for statements that contains fn or class.

Reply:
then you need to insert every function and class for  `__name`